lib/codecs/context.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+// map of future CBOR-LD registry of known JSON-LD Contexts
+const encodeMap = new Map();
+const decodeMap = new Map();
+_setEncodeDecodeMap(0x10, 'https://www.w3.org/ns/activitystreams');
```

Comment:
I assume there will be another registry for these ?

I'd prefer not to have another registry... and instead simply encode the string contexts next to the object.... tiny amount of extra size for the ability to eliminate a registry.

Reply:
@OR13, the goal is to have three features, any of which can be used, depending on use case:

1. A reference to a global registry. A tag will call out that you need to look in a global registry for tag => context URL mappings.
2. A reference to a user/local registry. A tag will indicate, instead, that you need to have the user/local registry to get the context URLs.
3. Directly including the context URL in the data. A tag will indicate that the full URL is directly available in the data, just use it.

We need the first two options to handle highly compressible CBOR-LD; it makes an important difference for very space-limited cases.

Reply:
I see, this comment is really just to restate what you have said:

1. if it's possible to compress a URL with a global registry, we do that.
2. if it's possible to use a local registry, we do that. 
3. in the case that there is no local registry or global one, the URL will be embedded